sample-values/values.aws.yaml

```diff
@@ -55,3 +55,4 @@ terminationGracePeriodSeconds: 30
 useCentralizedInfra: true
 usePersistentVolumes: true
 withSansSSL: true
+useCloudStorage: true
```

Comment:
which ckan-cloud-helm release introduced the use of this parameter?

Reply:
helm ckan v0.0.15

Reply:
Then I'm guessing you should use that version here, no? Otherwise this line doesn't have any effect.

Reply:
Ah, sure :)
Done.